packages/client/src/Config.ts

```diff
@@ -281,6 +281,13 @@ export interface StreamrClientConfig {
          * IP lookup).
          */
         location?: Location
+
+        /**
+         * Used to assign a custom external IP address for the node.
+         * Useful in cases where the node has a public IP address but
+         * the hosts network interface does not know of it
+         */
```

Comment:
Maybe we should add the limitation here for future reference.

```suggestion
         * Used to assign a custom external IP address for the node.
         * Useful in cases where the node has a public IP address but
         * the hosts network interface does not know of it.
         * 
         * Works only if the Full Cone NAT that the node is behind preserves local   
         * port mappings on the public side. 
         */
```